src/main/resources/application.properties

```diff
@@ -1 +1,21 @@
+server.port=8080
 
+#server.servlet.context-path=/categories/
+spring.application.name=vocabuilder
+
+spring.datasource.driver-class-name=org.h2.Driver
+spring.datasource.username=a
+spring.datasource.password=a
```

Comment:
do we need user and password for H2 DBs?

Reply:
I tried commenting username and password and I could not enter.